src/rs/scramble/megaminx.rs

```diff
@@ -0,0 +1,29 @@
+use cubing::alg::{Alg, AlgNode, Move, Newline};
+use rand::{thread_rng, Rng};
+
+const NUM_LINES: u8 = 7; // change this to USIZE later maybe
+const NUM_GROUPS: u8 = 5;
```

Comment:
I'll convert to `usize` after landing.

(We discussed this over video. `u8` is technically fine, I just personally prefer to stick with `usize` throughout the codebase as the default type for anything that represents an unsigned amount.)